[Form] Added support for caching choice lists based on options (#30994)
symfony/symfony: src/Symfony/Bridge/Doctrine/CHANGELOG.md

 * changed guessing of DECIMAL to set the `input` option of `NumberType` to string
 * deprecated not passing an `IdReader` to the `DoctrineChoiceLoader` when query can be optimized with a single id field
 * deprecated passing an `IdReader` to the `DoctrineChoiceLoader` when entities have a composite id

>  * added a `ChoiceList` facade to explicit choice lists caching based on callback options or loader when configured by specific form types

Reviewer: ```suggestion
 * added a `ChoiceList` facade for explicit choice lists caching based on callback options or loader when configured by specific form types
```

Author: This is not really the same meaning:
"I added something **to explicit** something when being something"
"I added something for **explicit something** when being something"

I would prefer to keep the first.

Reviewer 2: I would ask a native english speaker here :)

Reviewer: `explicit` is an adjective, not a verb. So you cannot `to explicit smth`
https://dictionary.cambridge.org/dictionary/english-russian/explicit

Author: Indeed. It is in French and wrongly translated as a verb here http://www.wordreference.com/fren/expliciter. I would propose `added a `ChoiceList` facade to make explicit choice lists caching...` then.

Reviewer: ```suggestion
 * added a `ChoiceList` facade to leverage explicit choice list caching based on callback options or specific loaders
```
